app/build.gradle.kts

```diff
@@ -130,6 +130,15 @@ kotlin {
     }
 }
 
+configurations.all {
+    resolutionStrategy.dependencySubstitution {
+        if (project.hasProperty("localSdk")) {
```

Comment:
Where is this supposed to get set and what kind of value is it supposed to have?

Reply:
@brian-livefront @david-livefront Just as an FYI, there are docs in our community contributor docs page: https://sdk.contributing-docs.pages.dev/getting-started/sdk/#linking-sdk-to-clients

I agree details need to be added in this project too. If not in `README`, then maybe in `docs/ARCHITECTURE.md`

Reply:
It doesn't really matter what value it has, since we just check if the property exists.

Reply:
I added a section to the readme under dependencies.